ui/apps/platform/cypress/integration/compliance/Compliance.helpers.js

```diff
@@ -115,7 +115,6 @@ function scanCompliance() {
     interactAndWaitForResponses(
         () => {
             cy.get(scanButton).click();
-            cy.get(scanButton).should('have.attr', 'disabled');
```

Comment:
I know there is a requirement listed in the [Jira ticket](https://issues.redhat.com/browse/ROX-21631) for this, but this seems unnecessary for performance and just to protect users from themselves. What does triggering another scan before seeing the results of the currently-running scan gain the user?

Reply:
I guess it's debatable, but my concern is that we have some long running scans that we have so far assumed will eventually finish and if not they will appropriately timeout. However, now with the current backend code and this code, if they do not finish, the user would never be able to trigger a new scan unless they bounce Central